src/main.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jan 25 18:37:33 2022
+
+@author: Phiala Thouvenin
+"""
+import glob
+import pims
+import pandas as pd
+import numpy as np
+from calc import *
+from util import *
+
+
+'''
+Generate particles for one experiment.
+Define basic parameters and generate synthetic particle grid
+'''
+# pjt, high friction, 15deg prebuilt taper, 9ero,glassbeads 1cm
+x_min, y_max = 140, 1220
+x_max, y_min = x_min+4400, 400
+# need to redefine as '../data/pjt_highfric...' if providing sample data
+prefix = 'pjt_highfric_15deg_1cmglassbeads_9ero'
+images = pims.ImageSequence(f'{prefix}/*.jpg')
+h5name = f'{prefix}/pjt_highfric_15deg_1cmglassbeads_9ero_trajs_size11.h5'
+surfs = f'{prefix}/pjt_highfric_15deg_1cmglassbeads_9ero_surfnocv.h5'
+xmin, xmax, ymin, ymax = 140, x_min+4400, 400, 1220
+pixelmm_scale = 3.05
+scale = 61.
+pfiles = glob.glob('pjt_highfric_15deg_1cmglassbeads_9ero/piv_nomask/*.txt')
```

Comment:
I hope I implemented `f'string {value}'` correctly - you can also use it here on line 31